start.sh

```diff
@@ -1,2 +1,2 @@
-aws sso login
-`aws configure export-credentials --format env`
+# aws sso login
```

Comment:
I don't know if this worked usefully/productively for others, but this never helped me so I opted to get rid of it.